src/main/java/io/jenkins/plugins/orka/helpers/OrkaInfoHelper.java

```diff
@@ -12,6 +12,7 @@
     private OrkaClientProxyFactory clientProxyFactory;
 
     private static final String[] supportedCPUs = new String[] { "3", "4", "6", "8", "12", "24" };
+    private static final String[] supportedSchedulers = new String[] { "", "default", "most-allocated" };
```

Comment:
I do not think this should be an empty string. 
We can call it `use-config-default` and provide a help text similarly to other settings explaining the three choices.

There is another approach.
Have a check box called `Use custom scheduler` or `Override scheduler`. If you check it you are presented with the choice between `default` and `most-allocated` if you do not check it, you use the config default.
Here we should have a helper text as well.

Reply:
![image](https://user-images.githubusercontent.com/35697323/158469455-92acf3a8-2fba-41cd-9f4d-3e218f363f2f.png)